qcoptions.py

```diff
@@ -8,7 +8,7 @@
                   help="Minimum value of scan parameter", metavar="scanmin", default=0)
 parser.add_option("--scanmax", type="int", dest="scanmax",
                   help="Maximum value of scan parameter", metavar="scanmax", default=254)
-parser.add_option("--vfatmask", type="int", dest="vfatmask",
+parser.add_option("--vfatmask", type="int", dest="vfatmask", 
```

Comment:
And since I'm making other change requests, prefer to remove all trailing whitespace (L11)